update.rb

```diff
@@ -50,7 +50,9 @@
 # Assignee to be set for this merge request.
 # Works best with marge-bot:
 # https://github.com/smarkets/marge-bot
-assignee = ENV["DEPENDABOT_ASSIGNEE_GITLAB_ID"]
+assignees = [ENV["DEPENDABOT_ASSIGNEE_GITLAB_ID"]].compact
```

Comment:
It is hard to review PRs in a language that I almost don't know! 😄 

Thanks! Today I learned about https://apidock.com/ruby/v1_9_3_392/Array/compact